Dockerfile

```diff
@@ -2,11 +2,14 @@ FROM python:3.8 as base
 RUN pip install poetry
 WORKDIR /app/
 COPY poetry.lock pyproject.toml /app/
-RUN poetry install --no-dev
+RUN poetry install 
```

Comment:
One alternative is to still take advantage of the `--no-dev` line here so that the production image only relies on production dependencies, but to add a separate `poetry install` step to the dev and test stages. It's more complex, but slightly nicer in terms of the final outputs